src/intervals/trigonometric.jl

```diff
@@ -27,15 +27,15 @@ end
 function sin{T}(a::Interval{T})
     isempty(a) && return a
 
-    whole_range = Interval(-one(T), one(T))
+    whole_range = Interval{T}(-1, 1)
```

Comment:
You were right to raise the issue (and I knew you would when I made the change ;) ). It led to a useful discussion. But I suggest that we leave the new version.